src/common/AlgoHelper.h

```diff
@@ -0,0 +1,204 @@
+#pragma once
+
+#include <algorithm>
+#include <functional>
+#include <random>
+#include <set>
+#include <stdexcept>
+#include <string>
+
+#include "faker-cxx/Datatype.h"
+#include "faker-cxx/Export.h"
+#include "faker-cxx/Helper.h"
+
+/**
+ * @brief Get a random element from a std::set.
+ *
+ * @tparam T an element type of the std::set.
+ *
+ * @param std::set of elements.
+ *
+ * @return T a random element from the std::set.
+ *
+ * @code
+ * std::set<char> chars{'a', 'b', 'c', 'd', 'e'};
+ * faker::helper::setElement(chars) // 'd'
+ * @endcode
+ */
```

Comment:
we don't need docs for internal functions, could be removed, applies to all of them in this file

Reply:
Job done as i think. i edited the file and removed comments as requested